pipelines/sqre/verify_drp_metrics.groovy

```diff
@@ -390,10 +381,9 @@ def void runDrpGen3(Map p) {
         util.bash '''
           set +o xtrace
           source /opt/lsst/software/stack/loadLSST.bash
-          setup -k -r "${LSST_VALIDATE_DRP_GEN3_CODE_DIR}/dustmaps_cachedata"
           set -o xtrace
 
-          "${LSST_CI_SCRIPTS_DIR}/run_validate_drp_gen3.sh"
+          "${LSST_CI_SCRIPTS_DIR}/run_verify_drp_metrics.sh"
```

Comment:
Just wondering: would it be easier to maintain this script here inline instead of having it be in ci_scripts?  I don't think anything else uses it.

Reply:
It's certainly not easier in the short term since that would require moving things around between packages.  I don't have a strong urge to change things just for the sake of it.